tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,17 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
-    result = vendor.remove(item)
+    result = False
 
-    raise Exception("Complete this test according to comments below.")
+    assert item not in vendor.inventory
+    assert result == False
```

Comment:
When the change on line 50 is reverted so that the test invokes `remove` you'll want to think about how can you make this test more robust.

Since the general intention of this method is to modify the inventory, we can check that the items that started in the inventory all remain there and also that inventory list is still 3 items long to make this test more robust.

```py
assert len(vendor.inventory) == 3
assert "a" in vendor.inventory 
assert "b" in vendor.inventory 
assert "c" in vendor.inventory

Reply:
✅ Fixed (will push later!)